src/main/java/ti4/map/Player.java

```diff
@@ -543,7 +543,7 @@ public boolean hasInf2Tech() {// "dszeliinf"
     @JsonIgnore
     public boolean hasWarsunTech() {
```

Comment:
Is this the method used to determine whether you're allowed to build a warsun?
Hm, can you see a way to just use the unitmodel for this? I.e. player.getUnitByBasetypeID("warsun") != Null? Future proofs any new warsun homebrew